resources/views/ingame/layouts/main.blade.php

```diff
@@ -123,16 +123,17 @@ class="overlay" data-overlay-title="My notes"
                     <li><a href="{{ route('logout') }}" onclick="event.preventDefault();
                                                      document.getElementById('logout-form').submit();">@lang('Log out')</a>
 
-                        <form id="logout-form" action="{{ route('logout') }}" method="POST" style="display: none;">
-                            {{ csrf_field() }}
-                        </form>
-                    </li>
-                    <li class="OGameClock">16.12.2017 <span>12:18:12</span></li>
-                </ul>
+                            <form id="logout-form" action="{{ route('logout') }}" method="POST"
+                                style="display: none;">
+                                {{ csrf_field() }}
+                            </form>
+                        </li>
+                        <li class="OGameClock"></li>
```

Comment:
This animation shows the top menu switching positions because of this empty initial date:
![fde1748e-2301-4fd4-bbd4-a29c8500fec4](https://github.com/lanedirt/OGameX/assets/6917405/fe22039e-96dc-476d-97ae-2ff743b81709)

I think this could be fixed by adding the initial time to the `<li class="OGameClock"></li>` element by using `date()`.